packages/sdk-react/src/components/approvalPolicies/ApprovalPolicyDetails/ApprovalPolicyView/utils/getVariantProps.ts

```
@@ -0,0 +1,139 @@
+      };
+    case 'cell':
+      return {
+        useBadge: false,
+        badgeVariant: 'secondary',
+        className: `mtw:text-sm mtw:font-medium mtw:text-foreground mtw:overflow-hidden mtw:text-ellipsis mtw:whitespace-nowrap mtw:block ${
+          showUsersAsEmail ? 'mtw:text-blue-600' : ''
+        } ${className}`,
+        showAvatar: false,
+        showIcon: false,
+      };
+    case 'text':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)